catalog/spatial/csw/spatial-csw-source/src/main/resources/OSGI-INF/metatype/metatype.xml

```diff
@@ -193,7 +195,8 @@
         </AD>
     </OCD>
 
-    <OCD name="GMD CSW Federated Source" id="Gmd_Csw_Federated_Source" description="GMD CSW Federated Source">
+    <OCD name="GMD CSW Federated Source" id="Gmd_Csw_Federated_Source"
```

Comment:
I meant to add ISO and the number to the description but forgot. Might be good to add for clarity. Your call